invokeai/app/invocations/controlnet_image_processors.py

```diff
@@ -593,14 +594,19 @@ def run_processor(self, image: Image.Image) -> Image.Image:
 
 
 DEPTH_ANYTHING_MODEL_SIZES = Literal["large", "base", "small"]
+DEPTH_ANYTHING_MODELS = {
+    "large": "LiheYoung/depth-anything-large-hf",
+    "base": "LiheYoung/depth-anything-base-hf",
+    "small": "depth-anything/Depth-Anything-V2-Small-hf",
+}
```

Comment:
Agreed. I'll add `small_v2` as a new option rather than replacing the current. Will also replace it as the new default in the Linear UI instead of Small V1.